setup.py

```diff
@@ -45,7 +45,7 @@ def find_package_data(where='.', package='', exclude=standard_exclude, exclude_d
+      long_description='# Sealing Juvenile Records\r\n\r\nThis process should be easier for both petitioners themselves and\r\nlegal professionals who hope to support petitioners. This tool\r\nis focused on MA.\r\n\r\n## Design\r\n\r\n### MVP should include\r\n\r\n1. Expectation setting.\r\n1. Links to support we cannot provide.\r\n1. Mobile friendly UI.\r\n1. A printable PDF.\r\n\r\n### Interview steps\r\n\r\n1. Check for immigration status **separately from the main interview**\r\n1. Screen for legal professionals\r\n1. Link to get certified dockets\r\n1. Sealing eligibility questions\r\n1. Personal details questions\r\n1. Downloadable/printable pre-filled petition PDF\r\n1. Provide the address for mailing the petition\r\n\r\n## Users\r\n\r\n### Target Audiences\r\n\r\n1. Petitioners.\r\n1. Legal professionals or clinic volunteers.\r\n\r\n### Unresovled Petitioner Barriers\r\n\r\n1. Frustration and distrust of the process.\r\n1. No access to printers.\r\n1. Which vocabulary will they find familiar?\r\n\r\n**User research needs to play a large role in the design of this tool.**\r\n\r\nThis tool is made with [Docassemble](https://github.com/jhpyle/docassemble).\r\n\r\n## Testing\r\n\r\n### Chai/Puppeteer tests\r\n\r\nThe first time you clone this repo, make sure you have NPM installed and run `npm install`.\r\nThis will install the libraries required for testing.\r\n\r\nTo run the tests, run `npm run test`.\r\n\r\n### Aloe/Lettuce tests\r\n\r\nTo get these tests to run, you need to download chromedriver from https://sites.google.com/a/chromium.org/chromedriver/ and put it in `tests/`\r\nYou also need to `pip3 install aloe selenium certifi`\r\n\r\nTo run [a feature test](https://docassemble.org/docs/development.html#bdd):\r\n```\r\ncd tests\r\naloe features/hello_world.feature\r\n```\r\n',
```

Comment:
Anyone know why this file got changed? I don't remember editing it for this PR. Or anytime recently.

Reply:
This should have been pulled in with the previous PR, I think.

Reply:
I think maybe the readme changes didn't get pulled on the last PR because I never went into playground?